config/dex/config.yaml

```diff
@@ -1,9 +1,6 @@
 issuer: http://127.0.0.1:5556
 storage:
-  config:
```

Comment:
what does this do? were we using a dex sqlite database befor?

Reply:
It's just for the tests. In general we do not use DEX at the moment.